### ISSUE TITLE
Crash when enter portal

### ISSUE THREAD

**Issue by uhloin** [Mysticpasta1/Dimension-Atlantis#69]
Crash report mentioned "xaero mod" but problem is in Atlantis because if I remove xaero mods then game still crashed when I try enter 
[crash-2022-11-02_18.37.47-client.txt](https://github.com/Exploding-Creeper/atlantis/files/9921422/crash-2022-11-02_18.37.47-client.txt)
portal.

Caused by: java.lang.NoSuchMethodError: 'float com.mystic.atlantis.entities.renders.JellyfishEntityRenderer.handleRotationFloat(net.minecraft.world.entity.LivingEntity, float)'

**Comment by Mysticpasta1** [comment]
Please test with Xaero mod and Atlantis only, we can't determine cause of the crash due to it being in a modpack

**Comment by uhloin** [comment]
OK, I tested.
Here are steps to reproduce from zero.
1. Get clean version of Minecraft 1.19.2
2. Install forge of version forge-1.19.2-43.1.47-installer.jar
3. Create mods subfolder with these versions of mods:
atlantis-2022.11.6-1.19.2-1.1.1-forge.jar
cloth-config-8.2.88-forge.jar
customportalapi-0.0.1-beta50-1.19.2.jar
geckolib-forge-1.19-3.1.36.jar 
MCglTF-1.19.2-Forge-2.0.0.0.jar 
Xaeros_Minimap_22.16.0_Forge_1.19.1.jar 
XaerosWorldMap_1.28.1_Forge_1.19.1.jar 
4. Create standard world in creative mode.
5. Find empty place and create on the grass horizontal square portal 4x4 with hole 2x2 from 12 athlantean core:
xxxx
x__x
x__x
xxxx
6. Pour water bucket to portal
7. Enter portal
In this clean version I could enter successfully and see some nice picture deep in ocean, tried move but after 2-3 seconds minecraft stops responding. Crush report is in attachment.
[crash-2022-11-08_01.19.04-client.txt](https://github.com/DimensionalDevelopment/atlantis/files/9955608/crash-2022-11-08_01.19.04-client.txt)

**Comment by uhloin** [comment]
May be you have different version of forge, I remember some mod was broken after changes with rendering events.
Also if this is important java version is jdk-17.0.2

**Comment by Mysticpasta1** [comment]
should be working that is weird, I will look into the bug

**Comment by uhloin** [comment]
I remember that I could enter portal for older version of mod and forge. But now (I checked) with older versions atlantis-2022.8.31-1.19.2-1.0.0-forge.jar and atlantis-2022.9.4-1.19.2-1.0.3-forge.jar Minecraft even cannot start. When It worked I entered portal and teleported to some dry cave under ocean floor. I digged out to sea water but get crush. May be I see some mob (jellyfish?) in water and its animation causes crush.
